Cloud Application Manager/Automating Deployments/version-control.md

```diff
@@ -1,82 +1,102 @@
 
 **Note:** This option is available for box versions only.
 
 ![version-control-and-auto-updates-4.png](../../images/cloud-application-manager/version-control-and-auto-updates-4.png)
 
-If you don’t set instances to auto-update, no worries. A visual cue on the Instances page reminds you when changes are available from the related box.
+Don't worry if you don’t set instances to auto-update. A visual cue on the Instances page reminds you when changes are available from the related box.
 
 ![version-control-and-auto-updates-5.png](../../images/cloud-application-manager/version-control-and-auto-updates-5.png)
 
-You can always manually update from the Instances page. Select the instance and under the Bulk Actions or the gear icon of the instance, click **Update Instance**.
+You can always manually update from Instances page. Select the instance and under the Bulk Actions or the gear icon of the instance, click **Update Instance**.
 
 ![version-control-and-auto-updates-6.png](../../images/cloud-application-manager/version-control-and-auto-updates-6.png)
 
 From here, you can update the instance with a specific version. An update always triggers a [reinstall operation](../Deploying Anywhere/deploying-managing-instances.md).
 
 ![version-control-and-auto-updates-7.png](../../images/cloud-application-manager/version-control-and-auto-updates-7.png)
 
-**Note:** When deploying a box from the Instances page, you consume the latest configuration of the parent and child boxes unless you chose a specific version for the child boxes. To deploy a different version of the parent box, launch it from the box page. From the Versions tab > gear icon, click **Deploy** on a select version.
+**Note:** When deploying a box from Instances page, you consume the latest configuration of the parent and child boxes unless you chose a specific version for the child boxes. To deploy a different version of the parent box, launch it from the box page. From the **Versions** tab > gear icon, click **Deploy** on a selected version.
 
-### Manage a version box
+### Managing box versions
 
-Version boxes can be managed in different ways. To see these options, go to versions tab and select gear icon.
+Box versions can be managed in different ways. To see these options, go to **Versions** tab and click the gear icon.
 
 ![version-control-and-auto-updates-8.png](../../images/cloud-application-manager/version-control-and-auto-updates-8.png)
 
-1. **Delete**: You can delete the selected version box. Please be aware that clicking on this option deletes the version without a warning popup.
-2. **Deploy**: You can deploy the selected version box.
-3. **Diff**: You can see the updates made to each version box from the versions tab. Select gear icon and click on **Diff**.
+1. **Delete**: You can delete the selected box version. Please be aware that clicking on this option deletes the version without a warning popup.
```

Comment:
Is this still true? Maybe a warning message was added...

Reply:
Yes, I checked it and that was still happening. Should I create a ticket for this?

Reply:
Yes, please, and also add to the ticket that this doc should be updated, ;)